### PR TITLE
 [Fit Import/Export] [Tour Nutrition] [Tour Blog] Multiple improvements and fixes

### PR DESCRIPTION
**Pull request by FJBDev**
- Fit Export: Fix for the Suunto App replay issue https://github.com/mytourbook/mytourbook/discussions/1194
- Fit Import: Code changes to support the upcoming https://github.com/mytourbook/mytourbook/discussions/1279  [Garmin Activity File Message Change](https://forums.garmin.com/developer/fit-sdk/b/news-announcements/posts/important-fit-activity-file-message-change) 
- Tour Nutrition: Product import nutrition values computation fix and product name adjustment (not concatenating the brands anymore)
- Cloud: Converting classes to [records](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Record.html)
- Tour Blog: Added the ability to display or hide the tour markers
image

### CODE REVIEW

File: bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java [hunk 1]
```diff
@@ -1465,6 +1471,7 @@ void updateUI() {
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java:1471]: Line 748: There should be space between the new tour summary and the tour title

image

File: bundles/net.tourbook.common/src/net/tourbook/common/util/ColumnManager.java [hunk 2]
```diff
@@ -3170,13 +3114,10 @@ private void setVisibleColumnIds_FromMenu(final MenuItem[] menuItems) {
 
          final Object itemData = menuItem.getData();
 
-         if (itemData instanceof ColumnDefinition) {
+         if (itemData instanceof final ColumnDefinition colDef) {
 
             final boolean isChecked = menuItem.getSelection();
 
-            // data in the table item contains the input items for the viewer
```
Comment (wolfgang-ch) [inline bundles/net.tourbook.common/src/net/tourbook/common/util/ColumnManager.java:3177]: This comment is removed

File: bundles/net.tourbook.common/src/net/tourbook/common/messages.properties [hunk 3]
```diff
@@ -85,7 +85,7 @@ Color_Chooser_SelectedColor_Tooltip               = Selected color in the color
 
 ColumnFactory_Category_Quirky = Quirky
 
-ColumnModifyDialog_Button_Default2_Tooltip           = Set columns to it's defaults (selection, sorting, width, ...)
```
Comment (wolfgang-ch) [inline bundles/net.tourbook.common/src/net/tourbook/common/messages.properties:88]: I was not aware about the differences between it's vs its 😄 

https://www.google.com/search?q=its+vs+it%27s

File: bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java:1471]: Fixed in this [PR](https://github.com/mytourbook/mytourbook/pull/1326)

File: bundles/net.tourbook.common/src/net/tourbook/common/util/ColumnManager.java (hunk 2 above)
Comment (FJBDev) [reply bundles/net.tourbook.common/src/net/tourbook/common/util/ColumnManager.java:3177]: Fixed in this [PR](https://github.com/mytourbook/mytourbook/pull/1326)

File: bundles/net.tourbook.common/src/net/tourbook/common/messages.properties (hunk 3 above)
Comment (FJBDev) [reply bundles/net.tourbook.common/src/net/tourbook/common/messages.properties:88]: Yes, both have a different meaning

**Pull request merged**